src/LiveComponent/src/Resources/doc/index.rst

```diff
@@ -1282,6 +1282,116 @@ and ``live_collection_button_delete`` respectively:
         {{ block('button_widget') }}
     {% endblock live_collection_button_add_widget %}
 
+If you only want to customize some attributes maybe simpler to use the options in the form type:
+
+    // ...
+    ->add('comments', LiveCollectionType::class, [
+        'entry_type' => CommentFormType::class,
+        'allow_add' => true,
+        'allow_delete' => true,
+        'by_reference' => false,
+        'label' => false,
+        'button_delete_options' => [
+            'label' => 'X',
+            'attr' => [
+                'class' => 'btn btn-outline-danger',
+            ],
+        ]
+    ])
+    // ...
+
+If you want more control over how each row is rendered you can override the blocks
+related to the ``LiveCollectionType``. This works the same way as for `the traditional
+collection type`_, but you should use ``live_collection`` and ``live_collection_entry`` as
```

Comment:
Did you mean `live_collection_widget` here?

Reply:
No, I just meant to mention it's just a prefix, it can be `live_collection_widget`, `live_collection_row`, `live_collection_errors`, ... and linked the docs for the collection field (https://symfony.com/doc/current/form/form_themes.html#fragment-naming-for-collections). Maybe better to write `live_collection_*` instead.